tests/test_gates.py

```diff
@@ -476,6 +514,63 @@ def circuit():
                 assert np.allclose(circ(), circ_def(), tol)
 
 
+@pytest.mark.parametrize("control_value", [False, True])
+@pytest.mark.parametrize("n_qubits", list(range(2, 8)))
+def test_controlled_sparse_qubit_unitary(n_qubits, control_value, tol):
+    """Test that a sparse ControlledQubitUnitary is correctly applied to a state"""
+
+    if device_name != "lightning.qubit":
+        pytest.skip("Skipping tests if not lightning.qubit.")
+
+    dev = qml.device(device_name, wires=n_qubits)
+    dev1 = qml.device(device_name, wires=n_qubits)
```

Comment:
Looks like we don't need this extra dev1

Reply:
Done!